dolphinscheduler-master/src/test/java/org/apache/dolphinscheduler/server/master/integration/cases/WorkflowStartTestCase.java

```diff
@@ -709,4 +709,30 @@ public void testStartWorkflowFromStartNodes_with_threeParallelSuccessTask() {
 
         assertThat(workflowRepository.getAll()).isEmpty();
     }
+
+    @Test
+    @DisplayName("Test start a workflow with one fake task success")
+    public void testStartWorkflow_with_oneSuccessFakeTask() {
+        final String yaml = "/it/start/workflow_with_one_fake_task_success.yaml";
+        final WorkflowTestCaseContext context = workflowTestCaseContextFactory.initializeContextFromYaml(yaml);
+
+        final WorkflowOperator.WorkflowTriggerDTO workflowTriggerDTO = WorkflowOperator.WorkflowTriggerDTO.builder()
+                .workflowDefinition(context.getWorkflows().get(0))
+                .project(context.getProject())
+                .runWorkflowCommandParam(new RunWorkflowCommandParam())
+                .build();
+        workflowOperator.manualTriggerWorkflow(workflowTriggerDTO);
+
+        await()
+                .atMost(Duration.ofMinutes(1))
+                .untilAsserted(() -> {
+                    Assertions
+                            .assertThat(workflowTriggerDTO.getProject().getName())
+                            .isEqualTo("MasterIntegrationTest");
+                    Assertions
+                            .assertThat(workflowTriggerDTO.getWorkflowDefinition().getName())
+                            .isEqualTo("workflow_with_one_fake_task_success");
+                });
```

Comment:
You need to add a new yaml and do assertion at the content of LogicFakeTask, the current assertion is meanless.

Reply:
> You need to add a new yaml and do assertion at the content of LogicFakeTask, the current assertion is meanless.

I have checked the IT case and found that there is no verification of built-in parameters. I have also looked at other writing methods, but I still don't know how to write this very well. I need your guidance or a demo, thank you!

Reply:
> You need to add a new yaml and do assertion at the content of LogicFakeTask, the current assertion is meanless.

done